simulation/traffic_simulator/include/traffic_simulator/api/configuration.hpp

```diff
@@ -33,6 +35,8 @@ struct Configuration
 
   bool auto_sink = true;
 
+  std::set<traffic_simulator::EntityType, EntityTypeComparator> sinkable_entity_type = {};
```

Comment:
Instead of adding a less generic type like EntityTypeComparator, change it to add comparison operator overloads to EntityType.